### PR TITLE
[IOTDB-4109] Optimize merge、flush、clear cache operation

### PR DESCRIPTION
**Pull request by RYH61**
The flush/merge/clearcache on local operation runs directly locally, no longer transmits the datanodeid to the confignode.

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/executor/ClusterConfigTaskExecutor.java [hunk 1]
```diff
@@ -270,60 +269,71 @@ public SettableFuture<ConfigTaskResult> setTTL(SetTTLStatement setTTLStatement,
...
-      // Get response or throw exception
-      if (tsStatus.getCode() == TSStatusCode.SUCCESS_STATUS.getStatusCode()) {
-        future.set(new ConfigTaskResult(TSStatusCode.SUCCESS_STATUS));
-      } else {
-        future.setException(new StatementExecutionException(tsStatus));
-      }
+      tsStatus = client.merge();
     } catch (IOException | TException e) {
       future.setException(e);
     }
```
Comment (CRZbulabula) [inline server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/executor/ClusterConfigTaskExecutor.java:286]: Why this block of code doesn't inside the if statement like flush and clearCache does?

File: docs/UserGuide/Maintenance-Tools/Maintenance-Command.md [hunk 2]
```diff
@@ -22,7 +22,7 @@
 # Maintenance Command
 ## FLUSH
 
-Persist all the data points in the memory table of the storage group to the disk, and seal the data file.In cluster mode, we provide commands to persist the specified storage group cache of a single node and persist the specified storage group cache of the cluster.
+Persist all the data points in the memory table of the storage group to the disk, and seal the data file.In cluster mode, we provide commands to persist the specified storage group cache of local node and persist the specified storage group cache of the cluster.
```
Comment (CRZbulabula) [inline docs/UserGuide/Maintenance-Tools/Maintenance-Command.md:25]:
```suggestion
Persist all the data points in the memory table of the storage group to the disk, and seal the data file. In cluster mode, we provide commands to persist the specified storage group cache of local node and persist the specified storage group cache of the cluster.
```

**Review by CRZbulabula — COMMENTED** [review]
PTAL~

File: docs/UserGuide/Maintenance-Tools/Maintenance-Command.md (hunk 2 above)
Comment (RYH61) [reply docs/UserGuide/Maintenance-Tools/Maintenance-Command.md]: fixed~

**Review by CRZbulabula — APPROVED** [review]
LGTM～

**Review by neuyilan — APPROVED** [review]
LGTM

**Pull request merged**